docs/node/migrations/zero-downtime-migrations/index.mdx

```diff
@@ -0,0 +1,135 @@
+- `Timestamp`: The time at which the migration was proposed.
+
+The Migration proposal needs to be approved by at least 2/3rd of the validators to start the migration process. The migration proposal will expire if the supermajority of the validators do not approve the proposal within `100800` blocks (roughly 7 days with a 6 second block interval). Once the migration proposal is approved, the network moves into migration mode after the specified activation period.
+
+### Submit Migration Proposal
+
+A validator operator can submit a migration proposal using the `kwil-admin migrate propose` command as shown below. Refer to the [Submit Migration Proposal](/docs/ref/kwil-admin/migrate/propose) documentation for more details.
+
+```bash
+kwil-admin migrate propose --activation-period <activationPeriod> --duration <migrationDuration> --chain-id <newChainID> --rpcserver /tmp/kwil.sock
+```
+
+:::note
+An off-chain communication is required among the validator operators to decide on the details of the migration proposal. One of the validators should submit the proposal to the network for approval. Only one proposal can be applied to the network during the lifetime of the network (per chain-id). Once the proposal is approved, all the other migration proposals will be rejected from the network and no new proposals can be submitted or approved.
+:::
+
+To check if the proposal is submitted successfully, we can get the list of pending migration proposals as shown below:
+
+```bash
+kwil-admin migration list --rpcserver /tmp/kwild.socket
+```
+This list will contain the proposal ID which is used by the validators to vote on the proposal.
+
+
+### Approve Migration Proposal
+
+To start the migration, the migration proposal needs the approval from at least a supermajority of the validators. The validators can submit their vote approval on the migration proposal using the below command.
+
+```bash
+kwil-admin migration approve <proposal-id>
+```
+
+Operators can check the status of the proposal and the votes information as shown below:
+
+```bash
+kwil-admin migration status <proposal-id>
+```
+
+:::note
+The migration proposal will get expired if the supermajority of the validators do not approve the proposal within `100800` blocks (roughly 7 days with a 6sec block interval). The proposal can be resubmitted after the expiration.
+:::
+
+## Activation Period
+
+Once the migration proposal is approved, the migration is set to start after the activation period. During this activation period, no new migration proposals can be submitted or approved. The activation period allows all network participants to prepare for the upcoming migration.
+
+## Migration Starts
+
+The network migration begins once the specified activation period (measured in blocks) has passed since the approval of migration proposal. The following sections provide a detailed breakdown of the events that take place on both the existing and the new network throughout the migration period.
```

Comment:
It might be helpful to have a copy of the above diagram, and highlight which phase in the process you are describing in each section. This will help paint the picture of what is occuring.